### PR TITLE
Support for Modifying instances state

### PR DESCRIPTION
**Pull request by nvnyale**

### CODE REVIEW

File: api/types.go [hunk 1]
```diff
@@ -723,3 +723,7 @@ func parseAssociationTargets(rawTgts []*ssm.Target) (tgts []AssociationTarget) {
 	}
 	return tgts
 }
+
+type Ec2ChangeInstanceStateInput struct {
```
Comment (tenyo) [inline api/types.go:727]: `Ec2InstanceStateChangeRequest`

**Review by tenyo — CHANGES_REQUESTED** [review]

File: api/types.go (hunk 1 above)
Comment (nvnyale) [reply api/types.go]: Renamed.

File: ec2/instances.go [hunk 2]
```diff
@@ -240,3 +242,42 @@ func (e *Ec2) GetInstanceVolume(ctx context.Context, id, volid string) (*ec2.Vol
 
 	return out.Volumes[0], nil
 }
+
+// ChangeInstancesState is used to start, stop and reboot a given instance
+func (e *Ec2) ChangeInstancesState(ctx context.Context, state string, ids ...string) error {
```
[the remark this answers is not in the document]
Comment (nvnyale) [reply ec2/instances.go]: Done.

**Review by tenyo — APPROVED** [review]
Looks great 👍

**Pull request merged**